package.json

```diff
@@ -78,7 +78,8 @@
     "react-test-renderer": "^18.2.0",
     "style-loader": "^3.3.1",
     "ts-jest": "^29.0.3",
-    "webpack": "^5.65.0"
+    "webpack": "^5.65.0",
```

Comment:
Wait why `webpack`?

Reply:
Why `webpack` is requird?

Reply:
am not sure I think it was added by mistake

Reply:
Please remove `webpack` if not required.